Utils/hdinsight-node-common/src/com/microsoft/azure/sparkserverless/serverexplore/SparkServerlessClusterProvisionCtrlProvider.java

```diff
@@ -75,7 +75,10 @@ public static int getCalculatedAU(int masterCores,
     }
 
     public Observable<Integer> getAvailableAU() {
-        return Observable.just(0);
+        return account.getJobDegreeOfParallelism()
+                .subscribeOn(Schedulers.io())
+                .map(jobDegreeOfParallelism -> account.getMaxDegreeOfParallelism() < jobDegreeOfParallelism ? 0
+                        : account.getMaxDegreeOfParallelism() - jobDegreeOfParallelism);
     }
```

Comment:
In the dialogue, call UpdateTotalAU first (although it might still return after updateAvailableAU) because availableAU depends on result of getTotalAU